dat/missions/shark/sh02_tapping.lua

```diff
@@ -18,6 +19,9 @@
    <faction>Traders Guild</faction>
    <faction>Za'lek</faction>
   </avail>
+  <note_done name="Operation Cold Metal">
```

Comment:
If we are checking for `diff.isApplied('collective_dead')`, it might be better to do something like use meta-nodes. Have something like "provides" and "requires". You could have the "Operation Cold Metal" "provide" the `collective_dead` meta-node and have "Sharkman Is Back" "require" the `collective_dead` node. It is a bit trickier, but might be more flexible.

Reply:
Sounds good.
I guess this simple stuff could be used when it's really simple, like when there is simply an event inbetween